ssh/tailssh/tailssh.go

```diff
@@ -1069,13 +1070,20 @@ func (ss *sshSession) run() {
 			ss.ctx.CloseWithError(err)
 		}
 	}()
+	var openOutputStreams atomic.Int32
+	if ss.stderr != nil {
```

Comment:
Getting nitpicky, but instead of repeating the conditional here, why not do an `.Add(1)` in each of the goroutines where you start the copy?

Reply:
setting this in the goroutines individually is racy as one goroutine might run before the other and potentially even run to completion.
e.g.
gr1 - Add(1)
gr1 - error copying
gr1 - Add(-1)
gr1 - Val == 0 => CloseWrite

gr2 - Add(1)
gr2 - error copying
gr2 - Add(-1)
gr2 - Val == 0 => CloseWrite